tests/virtctl/create_vm.go

```diff
@@ -411,3 +499,11 @@ func createAnnotatedSourcePVC(instancetypeName, preferenceName string) *k8sv1.Pe
 	pvc := libstorage.CreateFSPVC("vm-pvc-"+rand.String(5), util.NamespaceTestDefault, "128M", pvcLabels)
 	return pvc
 }
+
+func beCreated() gomegatypes.GomegaMatcher {
```

Comment:
Please avoid duplicating existing code. If it is not possible to import the matcher from `vm_test.go` directly, can you please extract it or at least make it public so `vm_test.go` and `create_vm.go` can use it?